generators/python-v2/base/src/project/PyprojectToml.ts

```diff
@@ -0,0 +1,210 @@
+import { PythonDependency } from "./PythonDependency";
+
+/**
+ * Represents a pyproject.toml file and generates a string representation of it. 
+ */
+export class PyprojectToml {
```

Comment:
can we add some tests for this in a test file called `PyProjectToml.test.ts` ? I think this class also might be better renamed as `PyProjectTomlGenerator.ts`